README.md

```diff
@@ -35,6 +58,36 @@ This shell script contains commands that build and runs the fala app.
 
 If you're not using `Docker Desktop`, you may want to have two terminals open.
 
+### PlayWright Testing with Docker
+
+Playwright has its own Docker container. Tests are not executed in the FALA Docker container but instead inside a 
+Playwright Docker container, which looks for FALA's URLs to test against.
+
+You will need the FALA Docker container running in order for the tests to work, as the Playwright Docker container 
+interacts with FALA's URLs.
+
+To spin up the Playwright Docker container, do the following:
+`docker-compose up -d --build playwright`
+
+The Playwright Docker container executes all Playwright tests within the playwright directory of the project. 
+Once the tests are complete, the container will shut itself down.
```

Comment:
So this was confusing. Is the intent that once you run this command `docker-compose up -d --build playwright` the test's run inside the Docker container?  I had to look at Docker Desktop to see the output of the test result (which failed).

Would be worth writing this out here.

Reply:
Correct. I've added more detail explaining this. Playwright Docker container looks for FALA docker containers URL (port), so it can run e2e tests against it. 

Again, I need to see how you've set up your project to understand the failing in connections.